app/models/refinery/authentication/devise/user.rb

```diff
@@ -8,7 +8,8 @@ class User < Refinery::Core::BaseModel
 
         extend FriendlyId
 
-        has_and_belongs_to_many :roles, join_table: :refinery_authentication_devise_roles_users
+#        has_and_belongs_to_many :roles, join_table: :refinery_authentication_devise_roles_users
```

Comment:
we shouldn't keep commented out code around as git will keep track of changes for us